docs/ingestion/tasks.md

```diff
@@ -155,6 +155,14 @@ For some task types, the indexing task can wait for the newly ingested segments
 |`segmentAvailabilityWaitTimeMs`|Milliseconds waited by the ingestion task for the newly ingested segments to be available for query after completing ingestion was completed.|
 |`recordsProcessed`| Partitions that were processed by an ingestion task and includes count of records processed from each partition.|
 
+
+#### Compaction Task Fields
+
+| Field               | Description                                           |
+|---------------------|-------------------------------------------------------|
+| `segmentsRead`      | # of segments read by a parallel compaction task      |
+| `segmentsPublished` | # of segments published by a parallel compaction task |
```

Comment:
```suggestion
|Field| Description |
|---|---|
| `segmentsRead`| Number of segments read by compaction task with more than 1 subtask.|
| `segmentsPublished`| Number of segments published by a parallel compaction task |
```

nit: Minimal spacing in the table format to match other tables in this doc. https://github.com/apache/druid/blob/master/dev/druid_intellij_formatting.xml#L77-L80 - this change in the intelliJ formatting makes it so that intelliJ doesn't auto-format these tables.

https://github.com/apache/druid/blob/master/dev/intellij-setup.md - more detailed instructions on setting up the IDE to avoid formatting diffs in patches.